Dockerfile

```diff
@@ -55,28 +55,32 @@ RUN echo "Building OpenSSL" \
     && tar xzf openssl-$OPENSSL_VERS.tar.gz && cd openssl-$OPENSSL_VERS \
     && ./Configure $OPENSSL_ARCH -fPIC --prefix=$PREFIX_DIR \
     && make depend \
-    && make && make install \
+    && make -j$(nproc) && make install \
     && cd .. && rm -rf openssl-$OPENSSL_VERS.tar.gz openssl-$OPENSSL_VERS checksums.txt
 
 #####################
 ### Builder stage ###
 #####################
 
+FROM getsentry/sentry-cli:1.47.1 AS sentry-cli
```

Comment:
Can we move this to a later stage and then use `:1` here, just to make sure we get all bugfix releases in here? That way we wouldn't prune build caches in cases sentry-cli changes, but still get updates.

Reply:
Can change to `:1`, but this should go before the next line, that sets the current build context to `semaphore-deps` (and we need it for the build itself).

Reply:
Regarding cache pruning: AFAIK we don't always pull for the latest image version, but let me confirm.